tasks/sync-granule/tests/sync_granule_test.js

```diff
@@ -186,8 +186,13 @@ test.serial('download Granule from FTP endpoint', async (t) => {
 
   t.context.event.config.collection.url_path = 'example/';
 
-  await validateConfig(t, t.context.event.config);
-  await validateInput(t, t.context.event.input);
+  try {
+    await validateConfig(t, t.context.event.config);
+    await validateInput(t, t.context.event.input);
+  } catch (error) {
+    console.log(error);
+    throw error;
```

Comment:
Seems like this is just for debugging? Do we want to keep it?

Reply:
c91081459